<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckFiles</class>
 <widget class="QMainWindow" name="CheckFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CheckFiles</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnChooseSource">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>WYBIERZ FOLDER OBSERWOWANY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChooseTarget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>WYBIRZ FOLDER DOCELOWY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>300</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineSource">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>301</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineTarget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>321</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="logBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>721</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
